### PR TITLE
Banner cards

### PR DESCRIPTION
**Pull request by skumawat1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #[WRLDBAEMS-44](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-44)

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://banner-cards--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
       'error',
       {
         '*': 4,
-        hero: 5,
+        'curated-card': 5,
```
Comment (thomsebastin) [inline .eslintrc.js:25]: Should be `curated-cards`??

File: component-definition.json [hunk 2]
```diff
@@ -150,6 +150,37 @@
...
+        {
+          "title": "Curated Cards",
+          "id": "curated-cards",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Curated Cards",
+                  "model": "curated-cards",
```
Comment (thomsebastin) [inline component-definition.json:163]: Just for my understanding. Is the `model` name required here?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

File: component-definition.json (hunk 2 above)
Comment (skumawat1) [reply component-definition.json:163]: @thomsebastin , It is required as its map the authoring to block name in blocks folder where JS and CSS code resides

File: .eslintrc.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ module.exports = {
       'error',
       {
         '*': 4,
+        'curated-card': 7,
```
Comment (skumawat1) [inline .eslintrc.js:25]: @thomsebastin  this will be 'curated-card' as we have more than 5 authoring fields under 'curated-cards' block

**Review by mkbansal1 — APPROVED** [review]

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**